pallets/nfts/src/lib.rs

```diff
@@ -1316,6 +1372,65 @@ pub mod pallet {
 			)
 		}
 
+		/// Approve collection items owned by the origin to be transferred by a delegated
+		/// third-party account. This function reserves the required deposit
+		/// `CollectionApprovalDeposit` from the `origin` account.
+		///
+		/// Origin must be Signed.
+		///
+		/// - `collection`: The collection of the item to be approved for delegated transfer.
+		/// - `delegate`: The account to delegate permission to transfer collection items owned by
+		///   the origin.
+		/// - `maybe_deadline`: Optional deadline for the approval. Specified by providing the
+		/// 	number of blocks after which the approval will expire
+		///
+		/// Emits `TransferApproved` on success.
+		///
+		/// Weight: `O(1)`
+		#[pallet::call_index(39)]
```

Comment:
I think this dispatchable should be placed at the end based on the call index

Reply:
Refactored in https://github.com/r0gue-io/pop-node/pull/387/commits/829b433ccf2730516f3d834e2cd749a9979b5c38